test/module/irohad/ordering/on_demand_ordering_gate_test.cpp

```diff
@@ -103,16 +116,20 @@ TEST_F(OnDemandOrderingGateTest, BlockEvent) {
   ON_CALL(*proposal, transactions())
       .WillByDefault(Return(txs | boost::adaptors::indirected));
 
-  EXPECT_CALL(*ordering_service, onCollaborationOutcome(round)).Times(1);
-  EXPECT_CALL(*notification, onRequestProposal(round))
+  EXPECT_CALL(*ordering_service,
+              onCollaborationOutcome(ordering::nextCommitRound(round)))
```

Comment:
Same issue as above with logic duplication in ordering init and ordering gate. This test did not know about nextCommitRound and nextRejectRound calls before. Please comment.